### PR TITLE
add admissionregistration to the list

### PR DESCRIPTION
**Pull request by caesarxuchao**
Fix https://github.com/kubernetes/kubernetes/issues/47686

The bug is introduced by https://github.com/kubernetes/kubernetes/pull/46800

Any suggestion on how to write a unit test? Or don't bother because the hardcoded list will be gone soon after 1.7?

### CODE REVIEW

File: cmd/kube-apiserver/app/aggregator.go [hunk 1]
```diff
@@ -179,24 +179,25 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
...
+	{Group: "certificates.k8s.io", Version: "v1beta1"}:           {group: 17300, version: 9},
+	{Group: "networking.k8s.io", Version: "v1"}:                  {group: 17200, version: 15},
+	{Group: "policy", Version: "v1beta1"}:                        {group: 17100, version: 9},
+	{Group: "rbac.authorization.k8s.io", Version: "v1beta1"}:     {group: 17000, version: 12},
+	{Group: "rbac.authorization.k8s.io", Version: "v1alpha1"}:    {group: 17000, version: 9},
+	{Group: "settings.k8s.io", Version: "v1alpha1"}:              {group: 16900, version: 9},
+	{Group: "storage.k8s.io", Version: "v1"}:                     {group: 16800, version: 15},
+	{Group: "storage.k8s.io", Version: "v1beta1"}:                {group: 16800, version: 9},
+	{Group: "apiextensions.k8s.io", Version: "v1beta1"}:          {group: 16700, version: 9},
+	{Group: "admissionregistration.k8s.io", Version: "v1alpha1"}: {group: 16700, version: 9},
```
Comment (caesarxuchao) [inline cmd/kube-apiserver/app/aggregator.go]: @deads2k does the score make sense?

Comment (liggitt) [inline cmd/kube-apiserver/app/aggregator.go]: matches other alpha versions, seems ok

**Comment by liggitt** [comment]
manual verification is sufficient... if we had the data to write a test to keep this list up to date automatically, we would have used that data instead of a hardcoded list in the first place. I expect it to be gone in 1.8

**Pull request merged**